app/javascript/components/CardColumn/card-column-header/card-column-header.jsx

```diff
@@ -4,11 +4,12 @@ import Textarea from 'react-textarea-autosize';
 import {addCardMutation} from '../operations.gql';
 import BoardSlugContext from '../../../utils/board_slug_context';
 
-const CardColumnHeader = ({kind}) => {
+const CardColumnHeader = ({kind, onCardAdded, onGetIdNewCard, currentUser}) => {
   const textInput = useRef();
   const [isOpened, setOpened] = useState(false);
   const [newCard, setNewCard] = useState('');
```

Comment:
probably it is newCardBody or newCardContent, because Card is expected to be an object